opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -65,7 +65,7 @@ def test_aggregate_delta(self):
         synchronous_sum_aggregation.aggregate(measurement(2))
```

Comment:
Changes to this file are just renaming of variables, not much review is needed here.